### PR TITLE
docs(components): Add animation page to docs

### PR DESCRIPTION
**Pull request by rebecca-li**
### Added

- Docs for animation

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/design/src/animation.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+name: Animation
+menu: Design
+route: /animation
+---
+
+# Animation
+### Patterns
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:8]: Should this be an h2?

File: packages/design/src/animation.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+**Emphasis**
+
+Emphasis is used to bring attention to something on already on the screen. For example, when you want to guide users to a particular part of the interface for onboarding or learning purposes.
+
+**Delighter**
+
+Delighters can be used occasionally to create surprise and delight, and can give personality to our brand. These can be used to reinforce success in an important milestones in a user's journey.
+
+### Timing
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:28]: same - h2?

File: packages/design/src/animation.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+| Name        | Value                  | Examples of use              |
+| :--------------- | :--------------------- | :--------------------- |
+| `--duration-fastest` | 100ms    | smaller elements exiting, small elements, state changes
+| `--duration-fast`   | 200ms    | larger elements exiting, small elements, state changes
+| `--duration-normal`            | 300ms   | modals, side panels
+| `--duration-slow`          | 400ms | large elements traversing across large area
+| `--duration-slowest`          | 500ms | page transitions
+
+### Easing
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:42]: Same - h2?

File: packages/design/src/animation.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Delighters can be used occasionally to create surprise and delight, and can give personality to our brand. These can be used to reinforce success in an important milestones in a user's journey.
+
+### Timing
+Timing is used for animation durations.
+
+Use judgement when determining the appropriate timing. Duration will depend on the size of the element and the distance it covers. Animations for larger elements or longer distances should be longer than smaller elements or shorter distances.
+
+
+| Name        | Value                  | Examples of use              |
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:34]: I really like the examples of use here

File: packages/design/src/animation.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+| :--------------- | :--------------------- | :--------------------- |
+| `--duration-fastest` | 100ms    | smaller elements exiting, small elements, state changes
+| `--duration-fast`   | 200ms    | larger elements exiting, small elements, state changes
+| `--duration-normal`            | 300ms   | modals, side panels
+| `--duration-slow`          | 400ms | large elements traversing across large area
+| `--duration-slowest`          | 500ms | page transitions
+
+### Easing
+| Name        | Value                  | Examples of use              |
+| :--------------- | :--------------------- | :--------------------- |
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:44]: These guidelines are great

File: packages/design/src/animation.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### Timing
+Timing is used for animation durations.
+
+Use judgement when determining the appropriate timing. Duration will depend on the size of the element and the distance it covers. Animations for larger elements or longer distances should be longer than smaller elements or shorter distances.
+
+
+| Name        | Value                  | Examples of use              |
+| :--------------- | :--------------------- | :--------------------- |
+| `--duration-fastest` | 100ms    | smaller elements exiting, small elements, state changes
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:36]: The way these are written, are these going to be animation "tokens" that can be used in React? Or are they CSS variables?

File: packages/design/src/animation.mdx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+menu: Design
+route: /animation
+---
+
+# Animation
+### Patterns
+
+Animations in Jobber fall under categories based on their purpose:
+
+**Reveal**
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:12]: If the other headers are h2, can these be h3?

File: packages/design/src/animation.mdx (hunk 4 above)
Comment (chris-at-jobber) [reply packages/design/src/animation.mdx:34]: If we are calling these "Timing" options, can we change the variables to `timing` ie `--timing-fast`?

File: packages/design/src/animation.mdx (hunk 6 above)
Comment (rebecca-li) [reply packages/design/src/animation.mdx:36]: I was thinking they would be like tokens, similar to what we are doing currently for spacing

File: packages/design/src/animation.mdx (hunk 3 above)
Comment (chris-at-jobber) [reply packages/design/src/animation.mdx:42]: Can we add a descriptor for the purpose of easing? Something like "Animations are linear by default, but this lends and uncanny robotic feeling to the motion of an element. Easing creates a more natural, pleasing transition between animation states."

File: packages/design/src/animation.mdx (hunk 7 above)
Comment (rebecca-li) [reply packages/design/src/animation.mdx:12]: Yup, sounds good. Will go in and change the headers!

File: packages/design/src/animation.mdx (hunk 4 above)
Comment (rebecca-li) [reply packages/design/src/animation.mdx:33]: Makes sense we use one or the other. Framer motion uses "duration" so I've renamed this section

File: packages/design/src/animation.mdx [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+menu: Design
+route: /animation
+---
+
+# Animation
+## Patterns
+
+Animations in Jobber fall under categories based on their purpose:
+
+###Reveal
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:12]: Need space between the ### and the heading 
image

File: packages/design/src/animation.mdx [hunk 9]
```diff
@@ -39,8 +41,27 @@ Use judgement when determining the appropriate durations for animations. This wi
 | `--duration-slowest`          | 500ms | page transitions
 
 ## Easing
-| Name        | Value                  | Examples of use              |
-| :--------------- | :--------------------- | :--------------------- |
-| `--ease-in` | cubic-bezier(0.4, 0, 1, 0.9, 0)    | for exiting elements that don’t fade out
-| `--ease-out`   | cubic-bezier (0, 0, 0.4, 1)    | entering elements
-| `--ease-both`            | cubic-bezier(0.5, 0, 0.4, 1)   | for elements that move from point to point
+
+Animations are linear by default, but this lends and uncanny robotic feeling to the motion of an element. Easing creates a more natural, pleasing transition between animation states to mimic the acceleration and deceleration of objects in the real world.
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:45]: typo: ...lends **an** uncanny...

File: packages/design/src/animation.mdx [hunk 10]
```diff
@@ -5,10 +5,12 @@ route: /animation
 ---
 
 # Animation
+
 ## Patterns
 
 Animations in Jobber fall under categories based on their purpose:
 
+
 ###Reveal
```
Comment (chris-at-jobber) [inline packages/design/src/animation.mdx:14]: Still need spaces between the ### and the text, like:

`###Reveal` needs to be `### Reveal`

**Comment by chris-at-jobber** [comment]
Other than the one typo and the ### formatting this LGTM 👍

**Review by chris-at-jobber — APPROVED** [review]
LGTM!

**Pull request merged**